packages/cli/create-strapi-app/bin/index.js

```diff
@@ -2,4 +2,25 @@
 
 'use strict';
 
-require('../dist/create-strapi-app');
+const { run } = require('../dist/index.js');
+const readline = require('readline');
+
+if (process.platform === 'win32') {
```

Comment:
Could you add a small comment to explain why it's necessary?

Reply:
this code was already here before I just moved it I don't recall why it was here to be honest (sth about ctrl+c on windows) 🤔

Reply:
It is, thought it might add some clarity :shrug: